model/src/main/proto/analytics.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+
+package model;
+
+option java_package = "org.oppia.app.model";
+option java_multiple_files = true;
+
+// Supported event actions for analytics logging
+enum EventAction{
+  // The undefined user action of an event
+  UNKNOWN_EVENT_ACTION = 0;
```

Comment:
The number convention that we have been following across all protos is that we start it will `1` instead of `0`. So is there any specific reason why it is kept `0`.
Technically it won't create a difference for its usage but keeping consistency across the app would be good.

Ditto everywhere else in this PR.

Reply:
I think that is the case with the messages and not the enums. [Reference](https://github.com/oppia/oppia-android/blob/8998c1505260c9a9e41f246695378d6df4601b83/model/src/main/proto/profile.proto#L102)

Reply:
Yeah that makes sense. Thanks for pointing it out.